### PR TITLE
Feature/nur 410 fed nav refinement

### PR DESCRIPTION
**Pull request by Mitso**

### CODE REVIEW

**Comment by Mitso** [comment]
Add Learn module section and article page breadcrumbs
Learn module article page links must be numbered 
Article title arrow cannot be linked externally

skitch

**Comment by Mitso** [comment]
@projectmushroom please review

File: nurseconnect/static/src/sass/components.enhanced/ui/_organism-paper.scss [hunk 1]
```diff
@@ -133,26 +129,30 @@
...
               background-position: 0 0;
-              width: 20px;
-              height: 20px;
+              width: rem(25);
+              height: rem(25);
               position: absolute;
               right: 0;
               bottom: 0;
-              top: 15px;
+              top: 15px rem(15);
```
Comment (projectmushroom) [inline nurseconnect/static/src/sass/components.enhanced/ui/_organism-paper.scss]: 15px should probably be removed here

**Review by projectmushroom — APPROVED** [review]
🍰👍 all good, one small comment

**Pull request merged**